woocommerce/wc-admin
# Fix the activity panel toggle not closing on click

Fixes #6662 

Since the `event` in this case is the `blur` and not the `click`, we need to check the `relatedTarget`.  The `relatedTarget` property doesn't always seem to be caught on certain browsers like Safari.

This PR adds in another state to check when a panel is closing (this appears to be a code smell with the panel toggling, but I'm hesitant to refactor the panels on something we cherry pick into a release).

This PR also fixes the height.

### Screenshots

#### Before
<img width="339" alt="Screen Shot 2021-03-25 at 4 46 01 PM" src="https://user-images.githubusercontent.com/10561050/112541287-98e34800-8d89-11eb-9598-a3fc0a2c9b06.png">


#### After
<img width="351" alt="Screen Shot 2021-03-25 at 4 45 36 PM" src="https://user-images.githubusercontent.com/10561050/112541249-8e28b300-8d89-11eb-884e-86601a96eac0.png">


### Detailed test instructions:

1. Navigate to a page with the Inbox panel.
2. Toggle the panel open.
3. Click "Inbox" again.
4. Make sure the panel closes.
5. Open the panel again.
6. Test that clicking outside the panel closes it.
7. Make sure toggling/switching between two panels is unaffected.
